apps/component-date/index.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+const summary = require('hof-behaviour-summary-page');
+
+module.exports = {
+  name: 'component-date',
+  baseUrl: '/component-date',
```

Comment:
does HOF use the `name` as the `baseUrl` or vice versa? I think only one of them is required when they are the same